svg.js.d.ts

```diff
@@ -1382,6 +1388,13 @@ declare module "@svgdotjs/svg.js" {
         target(): string;
     }
 
+
+    // ForeignObject.js
+    class ForeignObject extends Element {
+        constructor(node: HTMLElement, attrs?: any)
+        add(element: Dom, attrs?: any) : ForeignObject

```

Comment:
The `element` can actually be anything from a html string over a html node to a svg.js object.
Hard to type...
And attrs is completely wrong here. `add()` takes an optional position `i` as second parameter

Reply:
```suggestion
        add(element: Dom, i?: number): ForeignObject
```

I see there is a lot of mess going on. Have to dig deeper